src/mumble/ListenerLocalVolumeSlider.h

```diff
@@ -0,0 +1,29 @@
+// Copyright 2022 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_MUMBLE_LISTENERLOCALVOLUMESLIDER_H_
+#define MUMBLE_MUMBLE_LISTENERLOCALVOLUMESLIDER_H_
+
+class Channel;
+
+#include "VolumeSliderWidgetAction.h"
```

Comment:
I think in most of the code base, we have includes before forward-declarations :thinking: